packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -0,0 +1,108 @@
+import { Element, Component, Host, Prop, h, Fragment } from '@stencil/core';
+
+@Component({
+  tag: 'gcds-card',
+  styleUrl: 'gcds-card.css',
+  shadow: true,
+})
+export class GcdsCard {
+  @Element() el: HTMLElement;
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop({ reflect: true }) type: 'link' | 'action' = 'link';
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop({ reflect: true }) cardTitle!: string;
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop() titleElement: 'h3' | 'h4' | 'h5' | 'h6' | 'a' = 'a';
```

Comment:
How do you feel about changing this prop to `titleTag` to be more consistent with the other `tag` prop?

Reply:
When first creating the property I was going to go with `title-tag` but changed it so it won't be confused with the `tag` property already available with the `gcds-card`.

That being said, I am willing to change it to `title-tag` if we think that is a better name for the property.

Reply:
Mmh maybe that is an Amy question haha Let's keep as is and maybe check with Amy next week when she is back.